src/graphics/text.rs

```diff
@@ -306,11 +306,11 @@ pub struct Text {
 }
 
 impl Text {
-    pub fn from_cached<T: Clone + Into<Cached<Texture>>>(ctx: &mut Graphics, texture: T) -> Self {
-        Self::from_cached_with_capacity(ctx, DEFAULT_TEXT_BUFFER_SIZE, texture)
+    pub fn new(ctx: &mut Graphics, texture: impl Into<Cached<Texture>>) -> Self {
+        Self::new_with_capacity(ctx, DEFAULT_TEXT_BUFFER_SIZE, texture)
     }
 
-    pub fn from_cached_with_capacity<T: Clone + Into<Cached<Texture>>>(
+    pub fn new_with_capacity<T: Into<Cached<Texture>>>(
```

Comment:
The general convention for `new` methods which also set a capacity value is `with_capacity`, as used by `VecDeque`, `HashMap`, `Vec`, etc. so can we rename to `with_capacity`? Also, I'd like it if you changed the `T: Into<Cached<Texture>>` bound on this one into an `impl Into<Cached<Texture>>` argument as on `new` instead so that the type signatures are similar-looking, and also move the `capacity` to the end so that the first two arguments are the same types for both `new` and `with_capacity`.